pkg/controllermanager/apis/config/v1alpha1/types.go

```diff
@@ -112,6 +112,9 @@ type ControllerManagerControllerConfiguration struct {
 	// ManagedSeedSet defines the configuration of the ManagedSeedSet controller.
 	// +optional
 	ManagedSeedSet *ManagedSeedSetControllerConfiguration `json:"managedSeedSet,omitempty"`
+	// ShootState defines the configuration of the ShootState finalizer controller.
+	// +optional
+	ShootState *ShootStateControllerConfiguration `json:"shootState,omitempty"`
```

Comment:
Defaulting?

Reply:
Thanks for pointing the missing defaulting, instrumented the controller configuration to default to the `DefaultControllerConcurrentSyncs` value whenever missing. The changes are available in my latest updates.